Fixes #75 Add note about import (#79)
xmldom/xmldom: readme.md

console.info(nsAttr)
console.info(doc)
```

> Note: in Typescript and ES6 you can use the import approach, as follows:

Reviewer: ```suggestion
Note: a more modern import approach is possible in Typescript and ES6, as follows:
```

Author: I can make the changes. Had just been trained by other projects with the 'fixes' prefix to allow auto closing of tickets, but I understand each project is different. I'll need to make a new PR to remove that and close this one.

Reviewer: I went ahead with the merge, just moved your fixes note from the subject into the comments in the commit message. Your note is shorter so I just kept it as-is. Thanks again!
